scripts/release-merge-to-main.ts

```diff
@@ -71,7 +71,8 @@ async function ensureBranch(owner: string, repo: string, branchName: string, tag
     });
 
     if (existingBranch.object.sha !== sha) {
-    // Branch exists, but points at the wrong hash; update it.
+      // Branch exists, but points at the wrong hash; update it.
+      console.log(`Updating existing branch ${ owner }/${ repo }/${ ref } to new commit ${ sha }`);
```

Comment:
Can you include the old SHA? I don't expect to ever need that, but feels like the right thing to do if you ever want to do some forensic work on the workflow.

Reply:
Done